mw-webapp/package.json

```diff
@@ -132,5 +99,38 @@
     "vite": "5.2.11",
     "vite-plugin-eslint": "1.8.1",
     "vite-tsconfig-paths": "4.3.2"
+  },
+  "engines": {
```

Comment:
why did you move all this stuff below dependencies?)

Reply:
I needed it to build app in docker containers, otherwise build would be too complex

Reply:
no,  I just see that you moved lines from 27-59 to 103-134